documentation/API-onboarding.md

```diff
@@ -0,0 +1,75 @@
+# API onboarding and management in CAMARA
+
+## Introduction
+
+This document aims at describing a clear, consistent and transparent process for the onboarding (submission) and management (maintenance) of API proposals and APIs in the CAMARA Project.
+
+The process pipeline is depicted in the figure below.
+
+![API process pipeline](ressources/API_process_pipeline.png)
+
+All stages within this process pipeline are documented in the CAMARA API overview list. An example can be seen here:
+
+![API process pipeline](ressources/API_overview_list.png)
+
```

Comment:
The API overview should contain the API version that is tested with the code referenced
Small correction: ressources should be resources